src/server/__tests__/teardown/teardown.js

```diff
@@ -16,5 +16,6 @@ export default async function teardown(results) {
     }
     await r.getPoolMaster().drain();
   }
+  await new Promise(res => setTimeout(() => res(), 50));
```

Comment:
jest does a crummy job of flushing to stdout before exiting. if i give it an extra 50ms, then all my console.log statements come through